js/core/game.js

```diff
@@ -291,13 +292,14 @@ define(["dao", "globals", "ui", "core/freeAgents", "core/finances", "core/gameSi
         };
 
         for (t = 0; t < 2; t++) {
-            keys = ['min', 'fg', 'fga', 'fgAtRim', 'fgaAtRim', 'fgLowPost', 'fgaLowPost', 'fgMidRange', 'fgaMidRange', 'tp', 'tpa', 'ft', 'fta', 'orb', 'drb', 'ast', 'tov', 'stl', 'blk', 'pf', 'pts', 'ptsQtrs'];
+            keys = ['min', 'fg', 'fga', 'fgAtRim', 'fgaAtRim', 'fgLowPost', 'fgaLowPost', 'fgMidRange', 'fgaMidRange', 'tp', 'tpa', 'ft', 'fta', 'orb', 'drb', 'ast', 'tov', 'stl', 'blk', 'ba', 'pf', 'pts', 'ptsQtrs'];
             for (i = 0; i < keys.length; i++) {
                 gameStats.teams[t][keys[i]] = results.team[t].stat[keys[i]];
             }
             gameStats.teams[t].trb = results.team[t].stat.orb + results.team[t].stat.drb;
 
             keys.unshift("gs"); // Also record starters, in addition to other stats
+            keys.push("plusminus");
```

Comment:
Why is this defined here rather than in the keys array definition a few lines up?